core/src/test/java/com/linecorp/armeria/server/logging/ContentPreviewingServiceTest.java

```diff
@@ -71,16 +71,16 @@ protected void configure(ServerBuilder sb) throws Exception {
                            return HttpResponse.of(responseHeaders,
                                                   HttpData.ofUtf8("Hello " + aggregated.contentUtf8() + '!'));
                        }));
+            sb.decorator("/beforeEncoding", ContentPreviewingService.newDecorator(100));
             sb.decorator("/beforeEncoding", EncodingService.builder()
                                                            .minBytesToForceChunkedEncoding(1)
                                                            .newDecorator());
-            sb.decorator("/beforeEncoding", ContentPreviewingService.newDecorator(100));
             sb.service("/beforeEncoding", httpService);
```

Comment:
How about moving this sentence before `.decorator()` for readability?

Reply:
I fixed 👍